VRDR/IJEMortality.cs

```diff
@@ -1039,11 +1038,11 @@ public string AGE
         {
             get
             {
-                if (record.AgeAtDeath != null)
+                if (record.AgeAtDeath != null && !String.IsNullOrWhiteSpace(record.AgeAtDeath["value"]))
                 {
                     return record.AgeAtDeath["value"];
                 }
-                return "";
+                return "999";
```

Comment:
I think you need to return 999 if AGETYPE is 9 regardless of the value of record.AgeAtDeath.